scripts/zephyr_module.py

```diff
@@ -215,20 +220,52 @@ def main():
     cmake = ""
     sanitycheck = ""
 
+    dep_modules = []
+    sorted_modules = []
+    start_modules = []
+
     for project in projects:
         # Avoid including Zephyr base project as module.
         if project == os.environ.get('ZEPHYR_BASE'):
             continue
 
         meta = process_module(project)
         if meta:
-            kconfig += process_kconfig(project, meta)
-            cmake += process_cmake(project, meta)
-            sanitycheck += process_sanitycheck(project, meta)
+            section = meta.get('build', dict())
+            deps = section.get('depends', [])
+            if not deps:
+                start_modules.append( [project, meta] )
```

Comment:
Please fix whitespace nits here and elsewhere, removing spaces between ( and [, as well as between ] and ).

Reply:
Will do.
Maybe it's my eyes, or I'm using a too small font size, but I always have hard time reading `([text])` as the parenthesis and square brackets kind of float together, thus I find an extra space easing the reading.

Reply:
And changing to tuple helped with the `([text])` as well.